torch/tensor.py

```diff
@@ -414,6 +414,10 @@ def __hash__(self):
         return id(self)
 
     def __dir__(self):
+        if self.is_quantized:
+            warnings.warn('dir() of a quantized tensor is possibly wrong. '
+                          'Not all members are supported (yet). '
+                          'Use at your own disgression!')
```

Comment:
Please change to:
'Only a small subset of methods are supported for quantized tensors. '